src/OidcClient.php

```diff
@@ -28,8 +29,13 @@
 class OidcClient implements OidcClientInterface
 {
   /** OIDC configuration values */
-  protected ?array $configuration = null;
-  private ?string  $cacheKey      = null;
+  protected ?array $configuration       = null;
+  private ?string  $cacheKey            = null;
+  private ?string  $codeChallengeMethod = null;
+  private array    $pkceAlgorithms      = [
+      'S256'  => 'sha256',
+      'plain' => false,
+  ];
```

Comment:
```suggestion
  private const PKCE_ALGORITHMS = [
      'S256'  => 'sha256',
      'plain' => false,
  ];
```

As it is a real constant value, but this will require some updates through the class.